# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../expect/setpgid.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -urN expect-5.43.orig/exp_command.c expect-5.43/exp_command.c
--- expect-5.43.orig/exp_command.c	2004-08-20 19:18:01.000000000 +0200
+++ expect-5.43/exp_command.c	2005-08-19 13:12:09.512460250 +0200
@@ -526,7 +526,7 @@
 #ifdef SETPGRP_VOID
     (void) setpgrp();
 #else
-    (void) setpgrp(0,0);
+    (void) setpgid(0,0);
 #endif
 }
 
